ext/node/ops/sqlite/mod.rs

```diff
@@ -38,6 +38,9 @@ pub enum SqliteError {
   #[class(generic)]
   #[error("Invalid constructor")]
   InvalidConstructor,
+  #[class(generic)]
+  #[error("Expanded SQL text would exceed configured limits")]
+  InvalidExpandedSql,
```

Comment:
Could you try creating a test that does that?

Reply:
I don't know if there is a reliable way. A test would have to create a query string takes up all available memory.